File: src/programmers/codingtest_practice/FindPrimeNum.java
package programmers.codingtest_practice;

import java.util.ArrayList;
import java.util.Iterator;
import java.util.List;

public class FindPrimeNum {

	 public static int solution(int n) {
	      int answer = 0;
	      
	      List<Integer> list = new ArrayList<Integer>();
	      for(int i=1; i<=n; i++){
	          list.add(i);
	      }
	       
	      for(int i=2; i<=list.size(); i++){
	          for(int j=2*i; j<=n; j+=i){
	        	  for (Iterator<Integer> iter = list.iterator(); iter.hasNext(); ) {
	    	    	  Integer o = iter.next();
	    	    	  if(o == j) iter.remove();
	    	      }
	          }
	      }

	      list.remove(1);
	      answer = list.size();
	      return answer;
	  }
	
	public static void main(String[] args) {
		solution(10);
	}
}

File: programmers/코딩 기초 트레이닝/find_prime_num/find_prime_num.py
def solution(n):
    l = [0 for i in range(n+1)]
    for i in range(2, n+1) :
        for j in range(2, int(n**1/2)+1) :
            if(i*j > n): break
            l[i*j] = 1

    s = [i for i in l[2:n+1] if i == 0]
    return len(s)